src/jdk.compiler/share/classes/com/sun/tools/javac/resources/compiler.properties

```diff
@@ -234,8 +234,20 @@ compiler.err.switch.expression.no.result.expressions=\
     switch expression does not have any result expressions
 
 # 0: name
-compiler.err.call.must.be.first.stmt.in.ctor=\
-    call to {0} must be first statement in constructor
+compiler.err.call.must.only.appear.in.ctor=\
+    calls to {0}() may only appear within constructors
+
+# 0: name, 1: name
+compiler.err.redundant.superclass.init=\
+    redundant call to {0}() after {1}() has already been invoked
+
+# 0: name
+compiler.err.calls.not.allowed.here=\
```

Comment:
The compiler message key seems overly general. maybe `constructor.call.not.allowed.here` ?

Reply:
Agreed - will fix.